src/bus/channels.js

```diff
@@ -0,0 +1,241 @@
+import isPlainObject from 'lodash.isplainobject';
+
+import { formatChannelFromServer, formatChannelToServer } from '../utils/bus';
+
+/**
+ * @typedef {Object} MessageBusChannel
+ * @property {string} id UUID formatted ID
+ * @property {string} name
+ * @property {string} organizationId UUID of the organization to which the channel belongs
+ * @property {string} serviceId
+ */
+
+/**
+ * Module that provides access to message bus channels
+ *
+ * @typicalname contxtSdk.bus.channels
+ */
+class Channels {
+  /**
+   * @param {Object} sdk An instance of the SDK so the module can communicate with other modules
+   * @param {Object} request An instance of the request module tied to this module's audience.
+   * @param {string} baseUrl The base URL provided by the parent module
+   */
+  constructor(sdk, request, baseUrl) {
+    this._baseUrl = baseUrl;
+    this._request = request;
+    this._sdk = sdk;
+  }
+
+  /**
+   * Creates a new message bus channel
+   *
+   * API Endpoint: '/organizations/:organizationId/services/:serviceId/channels'
+   * Method: POST
+   *
+   * @param {Object} channel
+   * @param {string} channel.name
+   * @param {string} channel.organizationId UUID corresponding with an organization
+   * @param {string} channel.serviceId ID of a service
+   *
+   * @returns {Promise}
+   * @fulfill {MessageBusChannel} Information about the new channel
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.bus.channels
+   *   .create({
+   *     name: 'Channel 46',
+   *     organizationId: '28cc036c-d87f-4f06-bd30-1e78c2701064',
+   *     serviceId: 'abc123service'
+   *   })
+   *   .then((channel) => console.log(channel))
+   *   .catch((err) => console.log(err));
+   */
+  create(channel = {}) {
+    const requiredFields = ['name', 'organizationId', 'serviceId'];
```

Comment:
Can we look into `util`'ing this pattern up? We use it so many places!

Reply:
A helper function may good to put into `contxt-express`

Reply:
Oh wait, i'm not in an API project... I forgot what I was reading. yes some sort of util function would be nice